game/src/starter/Game.java

```diff
@@ -127,6 +131,10 @@ protected void frame() {
         manageEntitiesSets();
         getHero().ifPresent(this::loadNextLevelIfEntityIsOnEndTile);
         if (Gdx.input.isKeyJustPressed(Input.Keys.P)) togglePause();
+        if (Gdx.input.isKeyJustPressed(KeyboardConfig.DEBUG_ACTIVATE_DEACTIVATE.get())) {
```

Comment:
Sollte dieser nicht in `DEBUG_TOGGLE_KEY` umbenannt werden?

```suggestion
        if (Gdx.input.isKeyJustPressed(KeyboardConfig.DEBUG_TOGGLE_KEY.get())) {
```